src/tools/opensc-tool.c

```diff
@@ -82,18 +82,18 @@ static const char *option_help[] = {
 	"Prints the ATR bytes of the card",
 	"Prints the card serial number",
 	"Identify the card and print its name",
-	"Get configuration key, format: section:name:key",
-	"Set configuration key, format: section:name:key:value",
+	"Get configuration, e.g. section:name:key",
+	"Set configuration, e.g. section:name:key:val",
 	"Lists readers",
 	"Lists all installed card drivers",
 	"Recursively lists files stored on card",
-	"Sends an APDU in format AA:BB:CC:DD:EE:FF...",
+	"Sends an APDU (may need '-c default')",
 	"Uses reader number <arg> [0]",
 	"Does card reset of type <cold|warm> [cold]",
-	"Forces the use of driver <arg> [auto-detect; '?' for list]",
+	"Forces a card driver (use '?' for list)",
```

Comment:
I would clarify this a bit more with something like " By default, the driver is detected as usual with opensc".